python/tank/platform/engine.py

```diff
@@ -910,7 +910,7 @@ def register_command(self, name, callback, properties=None):
 
         - ``group_default`` - Boolean value indicating whether this command should represent the group
           as a whole. Setting this value to True indicates that this is the command to run and display
-          in applications that show a single button for each command group.
+          in applications that show a single representation for any command group.
```

Comment:
can we also add a sentence to explain what should be expected to happen when 1) there is more than one group default defined for a group and 2) when there is no group default defined for a group?